### PR TITLE
feat(voltage-init-parameters): automatically apply modifications according to parameter

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**
Add a new Tab for general parameters in voltage init parameters

### CODE REVIEW

File: src/components/dialogs/parameters/voltageinit/use-get-voltage-init-parameters.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { VoltageInitParam } from './voltage-init-utils';
```
Comment (FranckLecuyer) [inline src/components/dialogs/parameters/voltageinit/use-get-voltage-init-parameters.tsx:1]: Missing license header

File: src/components/dialogs/parameters/voltageinit/voltage-init-parameters-form.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import yup from '../../../utils/yup-config';
```
Comment (FranckLecuyer) [inline src/components/dialogs/parameters/voltageinit/voltage-init-parameters-form.tsx:1]: Missing license header

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Tests: OK

File: src/components/dialogs/parameters/voltageinit/use-get-voltage-init-parameters.tsx (hunk 1 above)
Comment (TheMaskedTurtle) [reply src/components/dialogs/parameters/voltageinit/use-get-voltage-init-parameters.tsx:8]: Fixed

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**